doc/module-developer/src/output/output-variable-dos-and-don-ts.tex

```diff
@@ -49,11 +49,9 @@ \subsection{Output Variable Naming Conventions}\label{output-variable-naming-con
 
 \textbf{Dimensional Nature.} This name element is used to identify the nature of the physical dimensions of the quantity being reported in words.~ This name element is nearly always needed. This name element is the final part of the name so that names end with terms that reflect the nature of the value being reported. However, the name element should not be the actual units.~ For example, an output that has units of degrees Celsius, the final word in the name should be ``Temperature.''~ Or an output for air flow that has units of kg/s, the final words in the name should be ``Mass Flow Rate.''~ When reporting on electricity use in Watts, the term ``Power'' is used; however most other uses of Watts use the term ``Rate.''
 
-\textbf{Units}. This name element is used to identify the units of the value being reported.~ ~Units of output variables are strictly SI, as used in EnergyPlus.~ Units are concatenated on the end of the name with square brackets and separated by a space and.~ Dimensionless quantities shall use empty brackets ``{[} {]}.''~ The preferred set of units is listed at the top of the Energy+.idd file. Units are always abbreviated.
+In some situations, a model may lend itself to generating a series of related output variables that depend on input.~ For example a surface heat transfer model may have results for numerous individual nodes or cells across the thickness of the surfaces.~ The number of nodes in the model will vary at runtime depending on the Construction.~ When this is the case, the standard approach used in EnergyPlus is to produce a series of output variables by enumerating the output variable name (rather than the key value) to generate a specific name for each instance.~ The generated name element for each instance in the series is appended to the end of the name and should end in a number. For example, the third node in a stratified tank model is called ``Chilled Water Thermal Storage Temperature Node 3 {[}C{]}.''~ This is done to facilitate wildcarding when postprocessing output.
```

Comment:
For reference, there's no need to keep using the tildes at the end of each sentence in future code.  It's valid LaTeX, and part of the translation, but not necessary.